nginx-mainline.yaml

```diff
@@ -223,6 +222,30 @@ subpackages:
           cp -r ./nginx-mainline/* ${{targets.contextdir}}/usr/src/nginx
           ls -latr ${{targets.contextdir}}/usr/src/nginx
 
+  - name: ${{package.name}}-config-compat
+    description: Nginx config compatibility with upstream image
+    dependencies:
+      runtime:
+        - nginx-mainline
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/etc/nginx/conf.d
+          install -m644 -D nginx.default.conf ${{targets.subpkgdir}}/etc/nginx/conf.d/
```

Comment:
I'm wondering if we can get away with just:
```suggestion
```

Reply:
It was failing without line 231 "no file / directory", even with nginx-mainline as a runtime dep, so thats why I added in line 231. For line 232,  you don't think we meed nginx.default.conf copied in now?

Reply:
Ah interesting, please disregard this comment then. It makes sense since each subpackage will have its own target directory.